src/main/java/seedu/duke/data/notebook/Notebook.java

```diff
@@ -55,8 +57,11 @@ public void deleteNote(int index) {
     /**
      * Removes a note from the timetable.
      */
-    public void deleteNote(String title) {
-        notes.removeIf(note -> note.getTitle().equals(title));
+    public void deleteNote(String title) throws SystemException {
+        final boolean isDeleted = notes.removeIf(note -> note.getTitle().equals(title));
+        if (!isDeleted) {
+            throw new SystemException(SystemException.ExceptionType.EXCEPTION_NOTEBOOK_NOT_EXIST);
+        }
     }
```

Comment:
Make the function return the boolean instead of throwing exception? Also, should the equals be equalsIgnoreCase?